.travis.yml

```diff
@@ -2,5 +2,8 @@ language: node_js
 node_js:
   - "6"
 script:
-- python populateFeaturesFiles.py
-- npm run eslint
+  - python generateFeedChanges.py
```

Comment:
yeah, the only reason this is here is to generate the FeedChanges file which is for the legacy watcher. I think when we choose to move to WebExtensions, this would definitely be part of that work, probably not this one.